docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestration framework to help data engineers deliver high-quality data. In Dagster, the objects made by data engineers are called assets.
```

Comment:
If we're going to define Dagster here, I think we should use the same definition that we use in the docs frontpage: "Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports."